Branch-SDK-TestBed/AndroidManifest.xml

```diff
@@ -58,6 +58,7 @@
             <meta-data android:name="io.branch.sdk.auto_link_request_code" android:value="@integer/AutoDeeplinkRequestCode" />
         </activity>
 
+        <meta-data android:name="io.branch.sdk.DisableDeviceIDCapture" android:value="false" /> <!-- Set to true to disable Android ID capture-->
```

Comment:
I don't like the idea of putting more and more configuration in the manifest. Can we make this a part of the Application class?